main/src/ca/uwaterloo/flix/language/phase/Weeder2.scala

```diff
@@ -1753,8 +1754,16 @@ object Weeder2 {
             loc = tree.loc)
           sctx.errors.add(error)
           Validation.Success(Expr.Error(error))
-        // Case: run expr with eff { handlers... }
-        case (expr, withs) => Validation.Success(Expr.TryWith(expr, withs, tree.loc))
+        // Case: run expr with ..
```

Comment:
Comment...

Reply:
it is just saying that we now have a non-empty sequence of with, same as the old comment

Reply:
Like run expr [with exprs]... ?